### PR TITLE
Create dzdt.md

### PR DESCRIPTION
**Pull request by mattols**

### CODE REVIEW

File: book/projects/project_readmes/dzdt.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+* Team member: [Matt Olson](url to their webpage)
+
+## Project Goals
+
+In addition to learning more about working with ICESat-2 data on the cloud, our goals are to explore
+permafrost and land surface change with crossover ATL06 data
+
+## Project Outcomes
+
+* The group has learned more about working with cloud-hosted data and vizualization technies with geodataframes
```
Comment (jomey) [inline book/projects/project_readmes/dzdt.md]: @mattols - Looks like we have a small typo here

File: book/projects/project_readmes/dzdt.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# dzdt
+
+## Team Members
+
+The following people contributed to our project throughout the week:
+* Project lead: [Joanmarie Del Vecchio](url to their webpage)
+* Helper: [Scott Henderson](url to their webpage)
```
Comment (scottyhq) [inline book/projects/project_readmes/dzdt.md]:
```suggestion
* Helper: [Scott Henderson](https://github.com/scottyhq)
```

File: book/projects/project_readmes/dzdt.md (hunk 1 above)
Comment (scottyhq) [inline book/projects/project_readmes/dzdt.md]:
```suggestion
* The group has learned more about working with cloud-hosted data and visualization technies with geodataframes
```

File: book/projects/project_readmes/dzdt.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+* Helper: [Shashank Bushan](url to their webpage)
+* Team member: [Matt Olson](url to their webpage)
+
+## Project Goals
+
+In addition to learning more about working with ICESat-2 data on the cloud, our goals are to explore
+permafrost and land surface change with crossover ATL06 data
+
+## Project Outcomes
+
```
Comment (scottyhq) [inline book/projects/project_readmes/dzdt.md:17]: * Created a GitHub Repository with Jupyter Notebooks detailing exploratory analysis at several areas of interest in the Arctic https://github.com/ICESAT-2HackWeek/dzdt

* Closed a SlideRule GitHub Issue related to raster sampling of new ArcticDEM v4.1 release (https://github.com/ICESat2-SlideRule/sliderule/issues/299#issuecomment-1674246119)

* Contributed to notes on 3D Geodetic transforms to inter-compare various elevation datasets https://github.com/ICESAT-2HackWeek/3D_CRS_Transformation_Resources

Comment (scottyhq) [inline book/projects/project_readmes/dzdt.md:17]:
```suggestion
## Project Outcomes

* Created a GitHub Repository with Jupyter Notebooks detailing exploratory analysis at several areas of interest in the Arctic https://github.com/ICESAT-2HackWeek/dzdt

* Closed a SlideRule GitHub Issue related to raster sampling of new ArcticDEM v4.1 release (https://github.com/ICESat2-SlideRule/sliderule/issues/299#issuecomment-1674246119)

* Contributed to notes on 3D Geodetic transforms to inter-compare various elevation datasets https://github.com/ICESAT-2HackWeek/3D_CRS_Transformation_Resources
```

File: book/projects/project_readmes/dzdt.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+* The group has learned more about working with cloud-hosted data and vizualization technies with geodataframes
+* Joanmarie wrote a script to acquire ATL06 crossovers for dzdt 
+
+## Future Efforts
+
+Have you built any great new collaborations you anticipate continuing after the Hackweek?
+Is your team planning to keep meeting to continue hacking or draft a conference abstract or proposal?
+Did you start doing some work that relates to your research and enables you to contribute to shared tools for working with ICESat-2 data products?
+Please share your hopes and plans for the future here!
```
Comment (scottyhq) [inline book/projects/project_readmes/dzdt.md]:
```suggestion
We will continue to investigate elevation changes from ICESat-2 only (repeat tracks and crossovers), as well as comparing ICESat-2 with other time-dependent elevation data (ArcticDEM, InSAR)

The team plans to stay connected via GitHub and Slack and share periodic updates. We think there is high potential for an AGU abstract in 2024 and even a manuscript! 
```

File: book/projects/project_readmes/dzdt.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# dzdt
+
+## Team Members
+
+The following people contributed to our project throughout the week:
+* Project lead: [Joanmarie Del Vecchio](url to their webpage)
```
Comment (jmdelvecchio) [inline book/projects/project_readmes/dzdt.md]:
```suggestion
* Project lead: [Joanmarie Del Vecchio](https://jmdelvecchio.github.io/)
```

File: book/projects/project_readmes/dzdt.md (hunk 1 above)
Comment (jmdelvecchio) [inline book/projects/project_readmes/dzdt.md]:
```suggestion
* The group has learned more about working with cloud-hosted data and visualization techniques with geodataframes
```

File: book/projects/project_readmes/dzdt.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+## Project Goals
+
+In addition to learning more about working with ICESat-2 data on the cloud, our goals are to explore
+permafrost and land surface change with crossover ATL06 data
+
+## Project Outcomes
+
+* The group has learned more about working with cloud-hosted data and vizualization technies with geodataframes
+* Joanmarie wrote a script to acquire ATL06 crossovers for dzdt
```
Comment (jmdelvecchio) [inline book/projects/project_readmes/dzdt.md]:
```suggestion
* Joanmarie wrote a script that ingests a GeoJSON area of interest (AOI) which then (1) queries the [MODIS daily snow cover product](https://nsidc.org/data/mod10a1/versions/6) via Earth Engine and filters for"low-snow" dates over the study area, (2) queries `sliderule` for ATL06 tracks for the same AOI and filters the track collection dates to only those "low-snow" dates, (3) finds overlapping footprints from remaining measurements and differences the heights, and (4) creates an interactive Earth Engine and geopandas map that displays imagery from the lowest-snow days along with calculated height differences as proof-of-concept and for studying feasibility. 

  to acquire ATL06 crossovers for dzdt 
```

File: book/projects/project_readmes/dzdt.md (hunk 4 above)
Comment (jmdelvecchio) [reply book/projects/project_readmes/dzdt.md]: Something we didn't talk about but as I've been thinking more about it - maybe we could try to get some digital discussion space or a Zoom with the [surfit](https://github.com/ICESAT-2HackWeek/surfit) group to have some sort of 'supergroup' discussion for people broadly interested in characterizing morphologies of non-ice surfaces

File: book/projects/project_readmes/dzdt.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
+# dzdt
+
+## Team Members
+
+The following people contributed to our project throughout the week:
+* Project lead: [Joanmarie Del Vecchio](url to their webpage)
+* Helper: [Scott Henderson](url to their webpage)
+* Helper: [Shashank Bushan](url to their webpage)
```
Comment (ShashankBice) [inline book/projects/project_readmes/dzdt.md]:
```suggestion
* Helper: [Shashank Bhushan](https://github.com/ShashankBice)
```

File: book/projects/project_readmes/dzdt.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+## Project Goals
+
+In addition to learning more about working with ICESat-2 data on the cloud, our goals are to explore
+permafrost and land surface change with crossover ATL06 data
+
+## Project Outcomes
+
+* The group has learned more about working with cloud-hosted data and vizualization technies with geodataframes
+* Joanmarie wrote a script to acquire ATL06 crossovers for dzdt 
+
```
Comment (ShashankBice) [inline book/projects/project_readmes/dzdt.md:20]: * Reported bugs in [sliderule example documentation](https://github.com/ICESat2-SlideRule/sliderule/issues/300) and [icepyx tutorial](https://github.com/ICESAT-2HackWeek/ICESat-2-Hackweek-2023/pull/82).

**Comment by scottyhq** [comment]
@mattols are you able to commit these suggestions? will plan to merge tomorrow!

File: book/projects/project_readmes/dzdt.md (hunk 8 above)
Comment (ShashankBice) [inline book/projects/project_readmes/dzdt.md:19]:
```suggestion
* Reported bugs in [sliderule example documentation](https://github.com/ICESat2-SlideRule/sliderule/issues/300) and [icepyx tutorial](https://github.com/ICESAT-2HackWeek/ICESat-2-Hackweek-2023/pull/82).
```

**Review by JessicaS11 — APPROVED** [review]

**Pull request merged**